Zend/tests/isset_array.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
```

Comment:
I didn't find a better place to add more tests for `isset()` with arrays - the existing test cases I saw were either way too long or weren't testing arrays.